staging/src/k8s.io/component-base/logs/json/json.go

```diff
@@ -120,10 +140,10 @@ func (l *zapLogger) Error(err error, msg string, keysAndVals ...interface{}) {
 	checkedEntry.Write(l.handleFields(keysAndVals, handleError(err))...)
 }
 
-// V return info logr.Logger with specified level
+// V return info logr.Logger  with specified level
 func (l *zapLogger) V(level int) logr.Logger {
 	return &zapLogger{
-		lvl: l.lvl - zapcore.Level(level),
+		lvl: logLvl(level),
```

Comment:
New logr version defined verbosity as additive.
```suggestion
		lvl: l.level + level,
```